feeds/npm/npm_test.go

```diff
@@ -98,7 +141,31 @@ func barVersionInfoResponse(w http.ResponseWriter, r *http.Request) {
 {
 	"name": "BarPackage",
 	"dist-tags": {
-		"latest": "0.4.0"
+		"latest": "0.4.0",
+		"next": "0.5.0-alpha"
+	},
+	"time": {
+		"created": "2021-03-22T13:45:16.000Z",
+		"0.4.0": "2021-03-22T13:45:16.000Z",
+		"0.5.0-alpha": "2021-05-11T17:23:02.000Z"
+	}
+}
+`))
+	if err != nil {
+		fmt.Println("Unexpected error during mock http server write: %w", err)
+	}
+}
+
+func bazVersionInfoResponse(w http.ResponseWriter, r *http.Request) {
+	_, err := w.Write([]byte(`
```

Comment:
Does this err not need checking to satisfy the linter?

Reply:
It is checked on line `172`

Reply:
Very worrying diff.....